pkg/types/aws/validation/platform.go

```diff
@@ -9,6 +9,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	"github.com/openshift/installer/pkg/types/aws"
+	"github.com/sirupsen/logrus"
```

Comment:
nit: this should be in the middle import block. We try to follow the convention

```
<stdlib>

<external>

<internal>
```

We should enforce this by unit tests :/

Reply:
Updated the import section for the format.